ui/README.md

```diff
@@ -19,4 +20,8 @@ streamlit run main.py
 
 Change the config.yaml file to fit your custom configuration.  
 The provided version from github is aligned with the demo mlcommon
```

Comment:
@eranra link to the mlcommons example folder in the repo

Reply:
fixed